packages/web-components/src/field/field.styles.ts

```diff
@@ -0,0 +1,102 @@
+import { css } from '@microsoft/fast-element';
+import {
+  borderRadiusMedium,
+  colorNeutralForegroundDisabled,
+  colorStrokeFocus2,
+  spacingHorizontalM,
+  spacingVerticalS,
+  spacingVerticalXXS,
+  strokeWidthThick,
+} from '../theme/design-tokens.js';
+import { display } from '../utils/display.js';
+
+/**
+ * Selector for the `disabled` state.
+ * @public
+ */
+const disabledState = css.partial`:is([state--disabled], :state(disabled))`;
+
+/**
+ * Selector for the `focus-visible` state.
+ * @public
+ */
+const focusVisibleState = css.partial`:is([state--focus-visible], :state(focus-visible))`;
+
+/**
+ * Selector for the `valueMissing` state.
+ * @public
+ */
+const valueMissingState = css.partial`:is([state--valueMissing], :state(valueMissing))`;
```

Comment:
For consistency should this be `[state--value-missing]`?

Reply:
Updated all validation states to use kebab-case (for attribute matching and styles/states).